doc/exprtk.tex

```diff
@@ -1,374 +1,374 @@
+have fractional components truncation will be performed. (eg: $1.67\rightarrow1$)\tabularnewline
+\hline 
+%begin{latexonly}
+ & \tabularnewline
 \end{tabular}
 
 \begin{tabular}{|c|p{0.8\textwidth}|}
-\hline
-FUNCTION& DEFINITION\\
-  \hline
-  %end{latexonly}
-\verb':='& Assign the value of x to y. Where y is a mutable string  or
- string range and x is either a string or a string  range. eg:
- \begin{enumerate}
-\item\verb'y := x'
-\item\verb"y := 'abc'"
-\item\verb'y := x[:i + j]'
-\item\verb"y := '0123456789'[2:7]"
-\item\verb"y := '0123456789'[2i + 1:7]"
-\item\verb"y := (x := '0123456789'[2:7])"
-\item\verb"y[i:j] := x"
-\item\verb"y[i:j] := (x + 'abcdefg'[8 / 4:5])[m:n]"
+\hline 
+FUNCTION & DEFINITION\tabularnewline
+\hline 
+%end{latexonly}
+: & Assign the value of x to y. Where y is a mutable string or string
+range and x is either a string or a string range. eg: 
+\begin{enumerate}
+\item \verb'y := x' 
+\item \verb"y := 'abc'" 
+\item \verb'y := x[:i + j]' 
+\item \verb"y := '0123456789'[2:7]" 
+\item \verb"y := '0123456789'[2i + 1:7]" 
+\item \verb"y := (x := '0123456789'[2:7])" 
+\item \verb"y[i:j] := x" 
+\item \verb"y[i:j] := (x + 'abcdefg'[8 / 4:5])[m:n]" 
 \end{enumerate}
-
-Note: For options 7 and 8 the shorter of the two ranges 
-will denote the number characters that are to be copied.\\
-\verb'+'&  Concatenation of x and y. Where x and y are strings or  
- string ranges. eg
- \begin{enumerate}
-\item\verb"x + y"
-\item\verb"x + 'abc'"
-\item\verb"x + y[:i + j]"
-\item\verb"x[i:j] + y[2:3] + '0123456789'[2:7]"
-\item\verb"'abc' + x + y"
-\item\verb"'abc' + '1234567'"
-\item\verb"(x + 'a1B2c3D4' + y)[i:2j]"
-\end{enumerate}\\
-\verb'+='& Append to x the value of y. Where x is a mutable string 
-and y is either a string or a string range. eg:
+Note: For options 7 and 8 the shorter of the two ranges will denote
+the number characters that are to be copied.\tabularnewline
+ & Concatenation of x and y. Where x and y are strings or string ranges.
+eg 
 \begin{enumerate}
-\item\verb"x += y"                  
-\item\verb"x += 'abc'"              
-\item\verb"x += y[:i + j] + 'abc'"
-\item\verb"x += '0123456789'[2:7]"
+\item \verb"x + y" 
+\item \verb"x + 'abc'" 
+\item \verb"x + y[:i + j]" 
+\item \verb"x[i:j] + y[2:3] + '0123456789'[2:7]" 
+\item \verb"'abc' + x + y" 
+\item \verb"'abc' + '1234567'" 
+\item \verb"(x + 'a1B2c3D4' + y)[i:2j]" 
 \end{enumerate}
-\\
-\verb'<=>'&  Swap the values of x and y. Where x and y are mutable   
-            strings.  (eg: \verb'x <=> y')\\
-  %begin{latexonly}
-  \hline
+\tabularnewline
++ & Append to x the value of y. Where x is a mutable string and y is either
+a string or a string range. eg: 
+\begin{enumerate}
+\item \verb"x += y" 
+\item \verb"x += 'abc'" 
+\item \verb"x += y[:i + j] + 'abc'" 
+\item \verb"x += '0123456789'[2:7]" 
+\end{enumerate}
+\tabularnewline
+\textless = & Swap the values of x and y. Where x and y are mutable strings. (eg:
```

Comment:
\verb'<=>'

Reply:
Lyx should not be touching the contents of a \verb or verbatim environment. This is bad!